pkg/helm/release/manager.go

```diff
@@ -33,7 +35,6 @@ import (
 	"k8s.io/helm/pkg/proto/hapi/services"
 	"k8s.io/helm/pkg/storage"
 	"k8s.io/helm/pkg/tiller"
-	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/resource"
```

Comment:
Does this mean we can get rid of `k8s.io/kubernetes` dependency?

Reply:
I had this thought too. We don't have a direct dependency on `k8s.io/kubernetes` anymore, but it is still a transitive dependency.

https://github.com/helm/helm/blob/99199c975236430fdf7599c69a956c6eb73b44e9/glide.yaml#L45-L46

Reply:
hmm yes, I see, but we now pin `k8s.io/kubernetes `  to `=1.12.3` but they pin to `release-1.12`, should we not reflect what they pin to?

Reply:
Interestingly, they pin other kubernetes package dependencies to 1.12.0 while allowing `k8s.io/kubernetes` to follow the release 1.12 branch.

For us, I'd say either way probably works. Pinning to 1.12.3 means we align better to operator-sdk overall. But theoretically, patch releases shouldn't break compatibility, so `release-1.12` would be fine too. I don't have a strong opinion either way.

Reply:
Theoretically pinning to `release-1.12` should be fine because they shouldn't change API's. controller-runtime pins to [a specific release](https://github.com/kubernetes-sigs/controller-runtime/pull/230#issuecomment-443881429) so I went with that approach, and we can guarantee that nothing will break in generated operators.